applications/machine_learning/configuration/thingy53_nrf5340_cpuapp/app_ZDebug.conf

```diff
@@ -180,3 +188,9 @@ CONFIG_PM_EXTERNAL_FLASH=y
 CONFIG_PM_EXTERNAL_FLASH_DEV_NAME="MX25R64"
 CONFIG_PM_EXTERNAL_FLASH_BASE=0x0
 CONFIG_PM_EXTERNAL_FLASH_SIZE=0x800000
+
+# FLASH simulator is enabled to prevent bus faults when mcumgr tries to access
```

Comment:
This is not a correct fix or acceptable workaround.
hi @nvlsianpu , why we need to enabled this fake flash? applications do not have space for that.

Reply:
Mcumgr makes call to the flash_area on its own. This is not somethin we can cover by any mcuboot hook.